ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -551,6 +551,23 @@ def equal(self, y, name=None):
     @with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
     def rad2deg(self, name=None):
         return paddle_frontend.Tensor(ivy.rad2deg(self._ivy_array))
+    
+    @with_supported_dtypes({"2.5.1 and below": (
+         "float16",
+         "float32",
+         "float64", 
+         "bool", 
+         "int32", 
+         "int64"
+        )
+      }, 
+     "paddle" 
+    )
+    def diff(self,y, n=1,axis=-1,prepend=None,append=None, name=None):
+        self.ivy_array = paddle_frontend.Tensor(
+            ivy.diff(self._ivy_array, _to_ivy_array(y))
+        ).ivy_array
+        return paddle_frontend.Tensor(self)
```

Comment:
This should return an array if I'm correct

Reply:
The documentation from paddle says it is Tensor